### ISSUE TITLE
Error while rendering Swipeout element

### ISSUE THREAD

**Issue by RocKhalil** [dancormier/react-native-swipeout#127]
I'm getting this error `ExceptionsManager.js:76 Warning: There is an internal error in the React performance measurement code. Did not expect componentDidUpdate timer to start while componentWillMount timer is still in progress for another instance.` when trying to render a list with Swipeout buttons on iOS.
When removing the Swipeout buttons, then render is working fine.
You can find [here](https://github.com/facebook/react/issues/6949) a discussion about the received error where a FB Team member is discussing it.
Is there any way to fix the error from your side or we should wait for React Native's new release ?

**Comment by gaearon** [comment]
The text you posted is an errant *warning* that is unrelated to some other error (that causes the crash). The errant warning is fixed in React 15.3.2 but if you had a crash, please keep investigating it, because it is not related to the warning message you posted.

**Comment by RocKhalil** [comment]
Here's the fix:
You should update the `index.js` file; replace: 
```var React = require('react-native')
var tweenState = require('react-tween-state')
var {PanResponder, TouchableHighlight, StyleSheet, Text, View} = React``` 
with:
```var React = require('react')
var ReactNative = require('react-native')
var tweenState = require('react-tween-state')
var {PanResponder, TouchableHighlight, StyleSheet, Text, View} = ReactNative```
